app/utils/estimate_prioritization/estimate_prioritization.py

```diff
@@ -121,6 +123,28 @@ def get_prioritized_estimates(estimates: pd.DataFrame,
             selected_estimates.append(get_pooled_estimate(study_estimates))
         else:
             selected_estimates.append(study_estimates)
-    
-    selected_estimate_df = pd.concat(selected_estimates, axis = 1).T.astype(estimates.dtypes.to_dict())
-    return selected_estimate_df
\ No newline at end of file
+
+    if pool:
+        selected_estimate_df = pd.concat(selected_estimates, axis=1).T.astype(estimates.dtypes.to_dict())
+    else:
+        selected_estimate_df = pd.concat(selected_estimates)
+
+    return selected_estimate_df
+
+
+# Gets prioritized estimates without pooling by a specified subgrouping variable
+# "subgroup_var" = subgrouping variable to refrain from pooling by
+def get_prioritized_estimates_without_pooling(estimates: pd.DataFrame,
+                                subgroup_var: str,
+                                filters: Union[tuple, None] = None,
+                                **kwargs):
+    normal_estimates = get_prioritized_estimates(estimates, filters = filters, **kwargs)
+    unpooled_filters = set(filters) if filters else set()
+    unpooled_filters = tuple(set.union(unpooled_filters,
+                                       set([lambda estimate: estimate['subgroup_var'] == subgroup_var])))
```

Comment:
do we want the variable to only be subgroup_var? I wonder whether we want to make this a bit more general. Can't think of a specific use case off the bat -- most of the time it is subgroup var -- but...

Reply:
I get that we want generality here, just thought that the original name proposed "donotpoolby" would be hard to interpret and confusing for future devs. I'll try to think of better nomenclature